client/signup/steps/user/index.jsx

```diff
@@ -135,6 +136,19 @@ export class UserStep extends Component {
 		}
 	}
 
+	componentDidUpdate() {
```

Comment:
Should this be in `componentDidMount` instead of on each prop update? Feels like we'd only want to potentially redirect on first mount.

Reply:
I tried that and it didn't quite work as expected for me. 
It seems that in some cases the component is already mounted and this part gets skipped.